src/events/helpers.ts

```diff
@@ -9,15 +9,8 @@ import {
 
 const encoder = new UuidEncoder('base62')
 
-export const changeTimeZone = (
-  date: Date,
-  timeZone = 'Europe/London'
-): Date => {
-  const dateWithTimeZone = new Date(
-    date.toLocaleString('en-GB', {
-      timeZone,
-    })
-  )
+export const changeTimeZone = (date: Date): Date => {
+  const dateWithTimeZone = new Date(date)
```

Comment:
This seems unrelated to the Explore London section?

Reply:
This was to fix the CI issue. The `toLocaleString` seems to give `Invalid Date` for some tests:
![image](https://user-images.githubusercontent.com/19141517/108434650-eecf4800-723f-11eb-9af9-66772ab5b427.png)

I've tried running the tests locally on a new clone of the repository and the tests seem to still fail?

https://user-images.githubusercontent.com/19141517/108434906-52597580-7240-11eb-8483-c93804798f69.mov

@egmcdonald Should I create a new ticket to try and resolve? Not sure if it is replicable on any one else's machine except it did seem to fail in CI as well.

Reply:
let's move this to slack